bin/setup.js

```diff
@@ -9,7 +9,19 @@ var isInstallIntoApp = /node_modules/.test(process.cwd())
 // This block only executes if Hoodie is installed with as a dependency
 if (isInstallIntoApp) {
   var pathToAppRoot = path.resolve('..', '..')
-  var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+
+  // log warning if package.json cannot be found (hoodiehq/hoodie#751)
+  try {
+    var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+  } catch (error) {
+    if (error.code !== 'MODULE_NOT_FOUND') {
+      throw error
+    }
+    log.warn('setup', 'Could not find package.json at ' + path.join(pathToAppRoot, 'package.json'))
+    log.warn('setup', 'You must manually set the start script in your app’s package.json to "hoodie" in order for "npm start" to work')
+
+    process.exit(0)
```

Comment:
I misread the `beforeExit`, sorry!

https://www.nczonline.net/blog/2014/02/04/maintainable-node-js-javascript-avoid-process-exit/ explains the issue.

https://github.com/nodejs/node-v0.x-archive/issues/8329#issuecomment-54778937 Suggests wrapping the `process.exit()` call in a `process.nextTick()`